### PR TITLE
feat: #16752 - Added lv_LV localization

### LINKED ISSUE #16752: feat: add Latvian localization
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Adding localization for Latvian

### What does the proposed API look like?
Localizations

### PR DESCRIPTION
**Pull request by Kirbo**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

Add localization for lv_LV
close #16752

### 💡 Solution

Added the localization for lv_LV

### 📝 Changelog

Added localization for lv_LV

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Added localization for lv_LV |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
feature branch please

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "prop-types": "^15.6.2",
     "raf": "^3.4.0",
     "rc-animate": "^2.5.4",
-    "rc-calendar": "~9.13.3",
+    "rc-calendar": "~9.14.0-4.4.4.0",
```
Comment (zombieJ) [inline package.json]: just `~9.14.0`

Comment (kirbo) [reply package.json]: Fixed

**Comment by kirbo** [comment]
@zombieJ I'm having issues pushing the changes into feature branch:
```
[16:36:13] Kirbo-MBP-2018:ant-design kirbo$ git push -u origin feature/16752-latvian-localization
Total 0 (delta 0), reused 0 (delta 0)
To github.com:kirbo/ant-design.git
 ! [remote rejected]       feature/16752-latvian-localization -> feature/16752-latvian-localization (cannot lock ref 'refs/heads/feature/16752-latvian-localization': 'refs/heads/feature' exists; cannot create 'refs/heads/feature/16752-latvian-localization')
error: failed to push some refs to 'git@github.com:kirbo/ant-design.git'
```

Any suggestions what's wrong/what should I do?

**Comment by zombieJ** [comment]
Click `edit` to change the `master` to `feature` on the issue side:
屏幕快照 2019-05-25 下午10 11 32

Maybe need fix some conflict if met.

**Comment by kirbo** [comment]
@zombieJ like this you mean?

File: package.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
     "rc-input-number": "~4.4.0",
     "rc-menu": "~7.4.12",
     "rc-notification": "~3.3.0",
-    "rc-pagination": "~1.18.0",
+    "rc-pagination": "~1.19.0",
```
Comment (afc163) [inline package.json]: `1.20.1`

**Comment by afc163** [comment]
You should checkout from feature branch and commit.

File: package.json (hunk 2 above)
Comment (kirbo) [reply package.json]: Done

**Comment by kirbo** [comment]
> You should checkout from feature branch and commit.

Done

**Comment by afc163** [comment]
图片

**Comment by kirbo** [comment]
@afc163 better?

**Review by zombieJ — APPROVED** [review]

**Pull request merged**